### PR TITLE
Fixed message in appointments page and adjusted display of date. Fixes Issue #311 and #312 

### PR DESCRIPTION
**Pull request by avinashupadhya99**

### CODE REVIEW

File: app/views/admin/appointments/index.html.erb [hunk 1]
```diff
@@ -5,16 +5,16 @@
 <div class="columns">
   <div class="col-12" style="justify-content: center;">
     <ul class="pagination">
-      <li class="page-item page-prev">
```
Comment (jim) [inline app/views/admin/appointments/index.html.erb:8]: I see we've already got some inline `style` tags in this template, but can you please move the new styles into a `scss` file?

**Review by jim — CHANGES_REQUESTED** [review]
Thanks for the contribution! I'd like to try to keep the styles in SCSS files, although I see we've already had some leak out into this template.

File: app/views/admin/appointments/index.html.erb (hunk 1 above)
Comment (avinashupadhya99) [reply app/views/admin/appointments/index.html.erb:8]: Sure, I will do that

Comment (avinashupadhya99) [reply app/views/admin/appointments/index.html.erb:8]: It is done. @jim

**Review by jim — APPROVED** [review]
Thanks for moving those styles!

**Pull request merged**